### PR TITLE
Release 2.4.5

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
This is generated by the following.
```
$ ruby --version
ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-darwin18]

$ jekyll --version
jekyll 4.0.0
```

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
cc @srowen , @cloud-fan , @HyukjinKwon , @gatorsmile , @viirya , @maropu , @wangyum , @dbtsai

File: js/downloads.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, source
 var packagesV10 = [hadoop2p7, hadoop3p2, hadoopFree, sources];
 
 addRelease("3.0.0-preview2", new Date("12/23/2019"), packagesV10, true);
-addRelease("2.4.4", new Date("08/30/2019"), packagesV9, true);
+addRelease("2.4.5", new Date("02/05/2020"), packagesV9, true);
```
Comment (dongjoon-hyun) [inline js/downloads.js:27]: 1. `02/05/2020` is when the vote passed here.
2. `2020-02-08` is the news date in `news/_posts/2020-02-08-spark-2-4-5-released.md`.

**Review by maropu — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you, @maropu .

**Comment by dongjoon-hyun** [comment]
I'll merge and prepare announce email. We can update the website as a followup (if needed).

**Pull request merged**